backend/solver/origuide/loader.py

```diff
@@ -59,70 +57,31 @@ def create_faces(vertices, edges, faces_vertices):
     return faces
 
 
-def main():
-    if len(sys.argv) != 2:
-        print('Usage: python main.py <fold_file>')
-        sys.exit(1)
-
-    fold_path = sys.argv[1]
+def solve_fold(fold_path):
     fold = read_fold(fold_path)
+    first_frame = fold.frames[0]
```

Comment:
Personally I'd hide this `[0]` usage behind some function on the fold. Maybe having a root_frame as a property is not a bad idea after all...